gulpfile.js

```diff
@@ -25,7 +25,7 @@ gulp.task('run', function () {
   nodemon({
     script: 'src/index.js',
     ext: 'js',
-    env: { 'DEBUG': '*' },
+    /** env: { 'DEBUG': '*' }, **/
```

Comment:
I agree. That is confusing -- I'll add a comment along with fixes for whatever other review items are added here (by you or @mrose17).
The explanation is in the last PR for now: https://github.com/brave/ledger/pull/8#discussion_r81400872